app/views/conferences/_conference_details.html.haml

```diff
@@ -34,6 +34,8 @@
               = link_to "My Proposals", conference_program_proposals_path(conference.short_title), class: 'btn btn-default'
             - elsif can? :new, conference.program.events.new
               = link_to "Submit Proposal", new_conference_program_proposal_path(conference.short_title), class: 'btn btn-default'
+            - if current_user && current_user.ticket_purchases.by_conference(conference).unpaid.any?
```

Comment:
Just to give the option to user to pay their unpaid ticket purchases of that conference. I'll add a link in physical ticket index page as well. 🙂